src/stream/index.js

```diff
@@ -11,8 +11,7 @@ export default function stream(body, url = "/mesos/api/v1") {
       "Content-Type": "application/json",
       Accept: "application/json"
     }
-  })
```

Comment:
Would that make sense, or add value as being optional. As in I can decide whether I should reconnect myself, or leave this to the lib?

Or could the lib also offer me the observable that does the backoff properly, instead of leaving it to the client application to do that?

Reply:
So there's a specific ask from the Mesos team to implement backoff based on the heartbeat interval. So you need to read the value from the first event. It is not straightforward to implement it in rxjs. Back then we've spent some time implementing the proper backoff, couldn't come up with a solution that would be clear to understand and decided with the obnoxious `retry(-1)` just to ship it already.

Now I see this retry to cause more problems and would rather remove it completely so that a users could implement what they need.

Reply:
Makes sense.